src/js_sync.hpp

```diff
@@ -241,7 +251,13 @@ class SessionClass : public ClassDefinition<T, WeakSession> {
         {"_overrideServer", wrap<override_server>},
         {"addProgressNotification", wrap<add_progress_notification>},
         {"removeProgressNotification", wrap<remove_progress_notification>},
+        {"addConnectionNotification", wrap<add_connection_notification>},
+        {"removeConnectionNotification", wrap<remove_connection_notification>},
+        {"isConnected", wrap<is_connected>},
```

Comment:
Why not a property instead of a method?

Reply:
Mostly because we already have the `connectedState` property and this is more of a "calculated value" depending on some internal properties. I can change it though if you think it makes more sense as a property?